stylesheets/_component.datatables.scss

```diff
@@ -46,17 +46,28 @@
 }
 // scss-lint:enable SelectorFormat
 
-.dt-buttons {
-    border-right: 1px solid #ccc;
+a.dt-button {
+    border-right: 1px solid color(border);
+    margin-right: $margin-base-vertical;
+}
+
+.dt-buttons > a,
+.dt-buttons > button {
     float: left;
-    margin-top: 11px;
-    margin-right: 10px;
-    padding-right: 6px;
+    margin-top: $margin-large-vertical;
+    padding: 0 $padding-base-horizontal;
+}
+
+.dt-buttons > button {
+    margin-right: $margin-base-vertical;
 }
 
-.dt-button {
+button.dt-button {
     background-color: transparent;
-    border: 0;
+    border-bottom: 0;
```

Comment:
Could we replace `border-bottom:0`, `border-left:0` and `border-top:0` with `border: 0`, then set the `border-right`?